db/migrate/20220915125556_create_likes.rb

```diff
@@ -0,0 +1,11 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.integer :authorId
+      t.integer :postId
+    
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- As mentioned above, kindly make sure Foreign key is included with the corresponding index 👍🏽